docs/reference/troubleshooting/faq.md

```diff
@@ -84,19 +84,18 @@ There are a few other distinctions, for example, gas metering will be different
 
 zkSync Era’s data availability layer is Ethereum. All ecosystem projects that build on zkSync Era will inherit the full security benefits of Ethereum.
 
-This is obviously a critically important topic for us, and we’re currently in the midst of a major review of zkSync Era’s security (including external audits, security contests, and overhauling and extending our bug bounty program).
+This is obviously a critically important topic for us, and the system has gone through several security audits and bug bounty programs. You can read more about [zkSync Era security in this section of the docs](./audit-bug-bounty.md).
```

Comment:
This makes it sound like we've tried multiple different bug bounty programs. Perhaps we should rephrase to emphasize that we have one, successful program?

```suggestion
This is obviously a critically important topic for us, and the system has gone through several security audits and maintains a very detailed bug bounty program. You can read more about [zkSync Era security in this section of the docs](./audit-bug-bounty.md).
```